Bio/Phylo/PAML/baseml.py

```diff
@@ -177,7 +177,7 @@ def run(self, ctl_file=None, verbose=False, command="baseml", parse=True):
         if self.tree is None:
             raise ValueError("Tree file not specified.")
         if not os.path.exists(self.tree):
-            raise IOError("The specified tree file does not exist.")
+            raise OSError("The specified tree file does not exist.")
```

Comment:
This should be another `FileNotFoundError` instance. I'll continue adding these comments on the same errors below to as sign-posts when updating.